composer.json

```diff
@@ -80,6 +80,7 @@
         }
     },
     "scripts": {
+        "serve": "yii serve",
```

Comment:
This branch based on `patch-2` branch?
https://github.com/yiisoft/yii-demo/pull/259